FHEM/00_SIGNALduino.pm

```diff
@@ -2330,7 +2331,7 @@ sub SIGNALduino_Parse_MU($$$$@)
 				if ($partData ne "") {		# wurde die regex gefunden?
 					$message_start=$-[0];
 				} else {
-					SIGNALduino_Log3 $name, 5, "$name: $length_str $nrRestart.restarting, regex ($regex) not found, aborting";
+					Log3 $name, 5, "$name: $length_str $nrRestart.restarting, regex ($regex) not found, aborting" if ($dummy);
```

Comment:
Bitte wieder zurück ändern,
damit hat man keine Chance mehr es in ein eigenes Filelog zu melden und wenn das Gerät kein Dummy ist, geht es nicht.